spec/rack/test/cookie_spec.rb

```diff
@@ -216,4 +216,10 @@
       last_request.cookies.should == { "value" => "1" }
     end
   end
+
+  it "allow symbol access" do
+    jar = Rack::Test::CookieJar.new
+    jar["value"] = "foo;abc"
+    jar[:value].should == "foo;abc"
+  end
```

Comment:
@scepticulous let's give it a few days first. If he doesn't seem to be around I'm fine with you closing this PR and rebasing the change on `master` with the test at the right spot.